packages/dev-infra/_common.sh

```diff
@@ -36,8 +39,80 @@ export_redis_env() {
   export REDIS_HOST="127.0.0.1:6380"
 }
 
-# Main entry point
-main() {
+pg_clear() {
```

Comment:
Since this action is quite destructive, I wonder if it would be worthwhile to enforce that the db name or user includes "test", or something along those lines.  Let me know what you think.

Reply:
This is already the case. `pg_clear` is called from `cleanup` only if the "service" used is `db_test` which must come from the `DB_TEST_POSTGRES_URL` env var (notice that the name contains TEST here)

Reply:
<img width="914" alt="Capture d’écran 2023-12-05 à 11 57 04" src="https://github.com/bluesky-social/atproto/assets/813661/73479d49-01cf-4879-ac48-9d6b725ab74a">

The following appears in the console.

Reply:
That should be good 👍